src/RenderWebGL.js

```diff
@@ -1313,9 +1313,122 @@ class RenderWebGL extends EventEmitter {
         }, null);
     }
 
+    /**
+     * Update a drawable's skin.
+     * @param {number} drawableID The drawable's id.
+     * @param {number} skinId The skin to update to.
+     */
+    updateDrawableSkinId (drawableID, skinId) {
+        const drawable = this._allDrawables[drawableID];
+        // TODO: vm's requests to drawableID that do not have a Drawable object.
```

Comment:
Can we make this todo comment (and the associated ones below) clearer in terms of what actually needs to be done?

Reply:
Alternatively, we should file an issue either here or in VM and note it in these todo comments as well as the one in the now deprecated `updateDrawableProperties` function.

Reply:
Sure.

For clarity: The text of this current comment is an abbreviation of the comment in updateDrawable. https://github.com/LLK/scratch-render/pull/469/files#diff-0e66131b35a2396bf8b007193ee7bc77R1437-R1442

        if (!drawable) {
            /**
             * @todo fix whatever's wrong in the VM which causes this, then add a warning or throw here.
             * Right now this happens so much on some projects that a warning or exception here can hang the browser.
             */
             return;

This is a very old issue. And not one these new updateDrawable* methods are introducing.